### ISSUE TITLE
t/07_token.t:  2 unit test failures in bleadperl as of commit 7259f419

### ISSUE THREAD

**Issue by jkeenan** [adamkennedy/PPI#245]
The following recent change in the Perl 5 core distribution has caused two unit tests in t/07_token.t to fail.
```
commit 7259f4194f3131957240f6b3dba47b74f53ac660
Author:     Tony Cook <tony@develop-help.com>
AuthorDate: Tue Jun 4 16:35:35 2019 +1000
Commit:     Tony Cook <tony@develop-help.com>
CommitDate: Wed Jun 12 09:52:00 2019 +1000

    (perl #134125) accept only complete hex and binary literals
```
As discussed in Perl issue [134210](https://rt.perl.org/Ticket/Display.html?id=134210), this emerged when the core corrected a previously reported bug.  Sometimes these changes expose problems in downstream test suites.  We would like to ask that PPI be examined to see how it can be adapted to cope with Perl 5 blead and, ultimately, perl-5.32.  PPI has a large number of downstream CPAN distributions, so a failure in its test suite prevents those distributions from being tested or installed in a dependency-order system.

Thank you very much.
Jim Keenan

**Comment by andk** [comment]
Xref: The "previously reported bug" was https://rt.perl.org/Ticket/Display.html?id=134125 and it seems like a still moving target

**Comment by tonycoz** [comment]
I've committed what I expect to be the final version to blead.

For this test failure it should only change the error message reported (the test only checks for failure, not the content of the error)

**Comment by wchristian** [comment]
fixed and released in v1.270
